mne/viz/_figure.py

```diff
@@ -630,7 +630,8 @@ def _keypress(self, event):
         else:
             last_time = self.mne.inst.times[-1]
         # scroll up/down
-        if key in ('down', 'up'):
+        if key in ('down', 'up', 'shift+down', 'shift+up'):
```

Comment:
Also to make it so that you can navigate full pages in up/down or left/right while holding shift (without needing to release it) I added an alias here for shift+down to move down